hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2197,20 +2234,25 @@ public void renameKey(OmKeyArgs args, String toKeyName) throws IOException {
    */
   @Override
   public void deleteKey(OmKeyArgs args) throws IOException {
+    Map<String, String> auditMap = args.toAuditMap();
```

Comment:
The same comment for all write requests:
Can we remove the changes from old write code path, which is not required? It is unnecessary now.

Reply:
I would have preferred removing old write code path before implementing links to avoid duplicate work.  However, now that these changes are in place, I prefer keeping them, and removing old write code path completely in a separate step.

Reply:
Makes sense to me.